pkg/infra/container/container.go

```diff
@@ -42,7 +42,7 @@ const (
 	RoleLabel           = "sealer-io-role"
 	RoleLabelMaster     = "sealer-io-role-is-master"
 	NetworkName         = "sealer-network"
-	ImageName           = "registry.cn-qingdao.aliyuncs.com/sealer-io/sealer-base-image:latest"
+	ImageName           = "sealerio/sealer-base-image:v1"
```

Comment:
What does this image do？

Reply:
I see that we use both `registry.cn-qingdao.aliyuncs.com/sealer-io/sealer-base-image:latest` and `sealerio/sealer-base-image:v1`. Can we unified them?

Reply:
> What does this image do？
This image is the base image used to simulate a node using a container and provides a sealer operating environment.

Reply:
> I see that we use both `registry.cn-qingdao.aliyuncs.com/sealer-io/sealer-base-image:latest` and `sealerio/sealer-base-image:v1`. Can we unified them?

`registry.cn-qingdao.aliyuncs.com/sealer-io/sealer-base-image:latest` is the image that sealer already had before, but there are some problems with it, we can abandon it. `sealerio/sealer-base-image:v1 ` is the image that I fixed the problems that the former has and it's Dockerfile is in `pkg/infra/container/imagecontext/base/Dockerfile`.